packages/host/app/services/matrix-service.ts

```diff
@@ -425,27 +426,7 @@ export default class MatrixService extends Service {
         });
         await realmSession.loaded;
         if (realmSession.canWrite) {
-          tools.push({
-            type: 'function',
-            function: {
-              name: 'patchCard',
-              description: `Propose a patch to an existing card to change its contents. Any attributes specified will be fully replaced, return the minimum required to make the change. If a relationship field value is removed, set the self property of the specific item to null. When editing a relationship array, display the full array in the patch code. Ensure the description explains what change you are making.`,
-              parameters: {
-                type: 'object',
-                properties: {
-                  card_id: {
-                    type: 'string',
-                    const: attachedOpenCard.id, // Force the valid card_id to be the id of the card being patched
-                  },
-                  description: {
-                    type: 'string',
-                  },
-                  ...patchSpec,
-                },
-                required: ['card_id', 'attributes', 'description'],
-              },
-            },
-          });
+          tools.push(getPatchTool(attachedOpenCard, patchSpec));
```

Comment:
@habdelra ur previous comment in my PR suggested to move this tool call somewhere. But, systematically now we use make a patch tool out of the context shared by the message. I would consider this out-of-scope for this PR as it requires notions of the `Command Card`. I understand the end-goal is to dynamically define tools for open ai.